code/index.html

```diff
@@ -1,30 +1,125 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Abel&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <section class="hero-image">
+      <div class="hero-container">
+        <h1>
+          <span>
+            FAMILJEN
+          </span>
+        </h1>
+        <span class="des">GO TO MORZINE WITH YOUR FIRENDS / OCTOBER 18 - 28 2020</span>
+      </div>
+    </section>  
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <section class="signup-container">
+      <div class="signup-text">
+        <span><p>Sign up for a masterclass in photography this fall. Lorem ipsum dolor sit amet consectetur adipisicing elit. Id maiores animi ipsam, laborum veniam placeat sapiente nulla incidunt veritatis, a corporis inventore deserunt impedit, fugit ex maxime odio officiis reiciendis.<p></span>
       </div>
+      <form action="http://httpbin.org/anything" method="POST">
+        <div class="form">
+          <input
+            id="firstName"
+            type="text"
+            name="firstNameValue" required
+          />
+          <label for="firstName" class="label-name">
+            <span class="content-name">First name</span>
+          </label>
+        </div>
```

Comment:
Really nice structure - keep it up! 
Also nice to see you use required for the input and connect the label by using 'for' correct 🏅